components/navbar/navbar.scss

```diff
@@ -1,142 +1,132 @@
-#burger {
-  margin-left: 1rem;
-  border: 0;
-  background: url(/assets/svg/hamburger.svg) no-repeat;
-  width: 31px;
-  height: 39px;
-}
+#main-nav-wrapper {
+  .burger {
+    margin-right: $value-28px;
```

Comment:
So since these variables are only used for things in `#main-nav-wrapper` we can declare the variables within the scope of its rule set.

```scss
#main-nav-wrapper {
  $burger-width: 28px;
  $burger-height: 28px;
  $burger-bar-offset: 13px;

  .burger {
    ...
    ...

}

```